docs/Configuration/Overview.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Configuration Overview
+permalink: /configuration/overview
+category: Configuration
+menuOrder: 1
+---
+
+Cube.js is designed to work with different configuration sources. There
+are two main ways you can set configuration options; via a configuration
+file, commonly known as the `cube.js` file; and environment variables.
+
+There are some areas of overlap between these options, and we recommend
+checking the configuration precedence. In general, defining options in
+your `cube.js` file is the most flexible option because you can define
+the options using JavaScript.
+
+## Configuration Precedence
+
+In Cube.js, environment variables take precedence over values specified
+in `cube.js`. The rationale behind this is simply due to the fact that
+it is generally easier and safer to switch environment variables for
+production than it is to do a new deployment.
+
+## Development vs Production
+
+Cube.js can be run in an insecure, development-only mode by setting the
+`CUBE_DEV_MODE` environment variable to `true`. Putting Cube.js in
+development mode does the following:
+
+- Disable authentication checks
+- Allows another log level to be set (`trace`)
+- Enables the Dev Server and Playground
+- Use `memory` instead of `redis` as the default cache/queue engine
+- Log incorrect/invalid configuration for `externalRefresh` /`waitForRenew` instead of throwing errors
```

Comment:
Some lines use infinitive forms of verbs ("disable"), other use Present Simple ("allows"). I suggest that we fight for uniformity that leads to clarity :)

Reply:
Great catch!